pkg/nvml/nvml.go

```diff
@@ -351,6 +351,15 @@ func nvmlDeviceClearCpuAffinity(nvmlDevice nvmlDevice) Return {
 	return __v
 }
 
+// nvmlDeviceGetNumaNodeId function as declared in nvml/nvml.h
+func nvmlDeviceGetNumaNodeId(nvmlDevice nvmlDevice, Node *uint32) Return {
```

Comment:
Question: Does this mean we don't need to use the heuristics that we currently use?

Reply:
I would say so. But we need to be aware that this is only available in later releases.

Reply:
Yes, we would have to support a fallback.